subcommand/connect-init/command.go

```diff
@@ -16,15 +17,22 @@ import (
 
 const bearerTokenFile = "/var/run/secrets/kubernetes.io/serviceaccount/token"
 const tokenSinkFile = "/consul/connect-inject/acl-token"
+const proxyIDFile = "/consul/connect-inject/proxyid"
 const numLoginRetries = 3
+const serviceRegistrationPollingRetries = 6 // This maps to 60 seconds
 
 type Command struct {
 	UI cli.Ui
 
-	flagACLAuthMethod   string            // Auth Method to use for ACLs, if enabled
-	flagMeta            map[string]string // Flag for metadata to consul login.
-	flagBearerTokenFile string            // Location of the bearer token.
-	flagTokenSinkFile   string            // Location to write the output token.
+	flagACLAuthMethod                  string            // Auth Method to use for ACLs, if enabled.
+	flagMeta                           map[string]string // Flag for metadata to consul login.
+	flagBearerTokenFile                string            // Location of the bearer token.
+	flagTokenSinkFile                  string            // Location to write the output token.
+	flagProxyIDFile                    string            // Location to write the output proxyID.
+	flagPodName                        string            // Pod name.
+	flagPodNamespace                   string            // Pod namespace.
```

Comment:
Open to ideas on addition of `flagPodName/space` , while it is unnecessary because we have these items in the ENV already it's a little bit easier to test this way.